package/utils/lua/Makefile

```diff
@@ -36,8 +38,8 @@ define Package/lua/Default
 endef
 
 define Package/lua/Default/description
- Lua is a powerful light-weight programming language designed for extending 
- applications. Lua is also frequently used as a general-purpose, stand-alone 
+ Lua is a powerful light-weight programming language designed for extending
+ applications. Lua is also frequently used as a general-purpose, stand-alone
```

Comment:
I know they are bad but can you drop these change from this commit? Or move them to another one? If in the future we will fix lto we can revert this commit